projects/extension/src/background/index.ts

```diff
@@ -28,14 +28,21 @@ export interface Background extends Window {
   }
 }
 
+type logStructure = {
+  time: string
+  level: string
```

Comment:
```suggestion
  level: number
```

Reply:
level is a number. I prefer to keep the description of the error on the logs